### DISCUSSION TITLE
Origin Trial for Topics

### DISCUSSION THREAD

**Discussion by lbdvt** [privacysandbox/privacy-sandbox-dev-support#36] [General] [answered]
Hi,

We are getting only a very low number of Topics (less than a hundred daily).

Is that expected and is the fix on the API still being rolled out for Chrome beta users?

Or should look for a bug on our end?

**Comment by samdutton** [comment, marked as answer]
Hi @lbdvt — do you mean you're getting a small number of topics returned by `document.browsingTopics()`, or a small number of users for whom the API is enabled (despite being on a page that provides a trial token)?

Topics is now at 50% of Beta users, but bear in mind that it will take some time for users to ramp up in the experiment and, once they're in the experiment it takes [at least a week](https://developer.chrome.com/docs/privacy-sandbox/topics/#how-does-the-topics-api-decide-which-callers-can-see-which-topic) to return topics.

**Reply by lbdvt** [reply]
Hi @samdutton,

As of today, we get a value from the Topics API less than 200 times per day, for around 50 users with the API enabled, surprisingly 94% of which happening on Chrome 102 (which is stable).

For comparison, we make ~1.1M Topics and FLEDGE API calls daily on Chrome 103-104-105, and we "tag" around 30k Interest Group daily.

**Reply by samdutton** [reply]
Just for anyone following this here — we followed up on this in a separate thread yesterday and will post back here with further details.

**Reply by lbdvt** [reply]
Hi @samdutton 

Thanks for your help! We fix an issue on our end, related to when tokens were provided on the page, and it seems to have fixed the issue.
This thread can now be closed.

**Reply by samdutton** [reply]
Good to hear — thanks @lbdvt!

I've marked this as answered.

[PR](https://github.com/GoogleChrome/developer.chrome.com/pull/3093) to update troubleshooting guide with your fix.
